src/client/features/search/index.js

```diff
@@ -34,12 +34,12 @@ class Search extends React.Component {
       dataSources: []
     };
 
-      ServerAPI.datasources()
-        .then(result => {
-          this.setState({
-            dataSources: Object.values(result)
-          });
-        });
+    ServerAPI.datasources()
+    .then(result => {
+      this.setState({
+        dataSources: Object.values(result).filter(ds => ds.hasPathways==true)
```

Comment:
`hasPathways` new in PC v10?

Reply:
No, it was always returned for each datasource in the array - by  /pc2/metadata/datasources query.
I just added this filed to the server-side pc code (https://github.com/PathwayCommons/app-ui/blob/issue_764_etc/src/server/pathway-commons/index.js#L107) and can filter as shown above.

PS:
note that internal pc client module is now just one file: src/server/pathway-commons/index.js, and there is no 'pathway-commons' import from npm.